Resources/Storage/StorageElement.py

```diff
@@ -397,39 +395,72 @@ def getOccupancy(self, unit='MB', **kwargs):
     filteredPlugins = self.__filterPlugins('getOccupancy')
     if not filteredPlugins:
       return S_ERROR(errno.EPROTONOSUPPORT, "No storage plugins to query the occupancy")
+
+    # Call occupancy plugin if requested
+    occupancyPlugin = self.options.get('OccupancyPlugin')
+    if occupancyPlugin:
+      res = ObjectLoader().loadObject(occupancyPlugin)
+      if not res['OK']:
+        return S_ERROR(errno.EPROTONOSUPPORT, 'Failed to load occupancy plugin %s' % occupancyPlugin)
+      log.verbose('Use occupancy plugin %s' % occupancyPlugin)
+      occupancyPlugin = res['Value'](self)
+      if occupancyPlugin.isUsable:
```

Comment:
I am not super keen in those *status* attribute, because it forces all the plugins to implement it, and they are not reliable. How about dropping that, and just surrounding the overall plugin section with a `try...except` block ?

Reply:
Also, please forward `kwargs` to your getOccupancy call such that any parameter can be given

Reply:
OK, I have implemented the features.